js/refresh.js

```diff
@@ -0,0 +1,5 @@
+var user = parameters['user'];
```

Comment:
I do not know where parameters is from and when it is present. If it is from other JavaScript files, it may not be present here but surely when the document is loaded.